README.md

```diff
@@ -45,6 +45,61 @@ This project offers a convenient method to host Huly using `docker`, designed fo
   - [Docker Compose](https://docs.docker.com/compose/install/)
 - Make sure what docker and `docker compose` commands are available in your terminal (e.g. `docker --version` and `docker compose --version`).
 - Make sure what docker and `docker compose` commands can be executed without sudo (e.g. `docker run hello-world` and `docker compose --version`).
+## Installing Node.js:
+Node.js is a JavaScript runtime that allows you to run JavaScript code outside of a web browser.
+
+
+### Via Package Manager:
```

Comment:
I would recommend installing it via `nvm`: https://nodejs.org/en/download/package-manager
We have `.nvmrc` in the project so this approach guarantees the required Node.js version will be installed.